### PR TITLE
More cleanups of main man page kpod-1.md and README.md

### PR DESCRIPTION
**Pull request by rhatdan**
Also added new commands to transfer.md

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,21 +39,24 @@ It is currently in active development in the Kubernetes community through the [d
...
+| [kpod(1)](/docs/kpod.1.md)                           | Simple management tool for pods and images                                                           |
+| [kpod-cp(1)](/docs/kpod-cp.1.md)                     | Copy files/folders between a container and the local filesystem                                      |
+| [kpod-diff(1)](/docs/kpod-diff.1.md)                 | Inspect changes on a container or image's filesystem                                                 |
+| [kpod-export(1)](/docs/kpod-export.1.md)             | Export container's filesystem contents as a tar archive                                              |
+| [kpod-history(1)](/docs/kpod-history.1.md)           | Shows the history of an image                                                                        |
+| [kpod-images(1)](/docs/kpod-images.1.md)             | List images in local storage                                                                         |
+| [kpod-info(1)](/docs/kpod-info.1.md)                 | Display System Information                                                                           |
```
Comment (TomSweeneyRedHat) [inline README.md]: system information?  (lower)

File: README.md [hunk 2]
```diff
@@ -39,21 +39,24 @@ It is currently in active development in the Kubernetes community through the [d
...
+| [kpod-pull(1)](/docs/kpod-pull.1.md)                 | Pull an image from a registry                                                                        |
+| [kpod-push(1)](/docs/kpod-push.1.md)                 | Push an image to a specified destination                                                             |
+| [kpod-rmi(1)](/docs/kpod-rmi.1.md)                   | Removes one or more images                                                                           |
+| [kpod-save(1)](/docs/kpod-save.1.md)                 | Saves an image to an archive                                                                         |
+| [kpod-tag(1)](/docs/kpod-tag.1.md)                   | Add an additional name to a local image                                                              |
+| [kpod-umount(1)](/docs/kpod-umount.1.md)             | Unmount a working container's root filesystem                                                        |
+| [kpod-version(1)](/docs/kpod-version.1.md)           | Display the Kpod Version Information                                                                 |
```
Comment (TomSweeneyRedHat) [inline README.md]: version information (lower)

File: docs/kpod.1.md [hunk 3]
```diff
@@ -46,17 +46,50 @@ has the capability to debug pods/images created by crio.
...
-Add one or more additional names to locally-stored image
+Add an additional name to a local image
 
-### info
-Displays system information
+### umount
+Unmount a working container's root file system
+
+### version
+Display the KPOD Version Information
```
Comment (TomSweeneyRedHat) [inline docs/kpod.1.md]: version information (lower)

**Comment by rhatdan** [comment]
@runcom As soon as these are reviewed, please merge, since no reason to run tests.

File: transfer.md [hunk 4]
```diff
@@ -39,11 +39,16 @@ There are other equivalents for these tools
...
-| `docker images` | [`kpod images`](./docs/kpod-images.1.md) |
-| `docker rmi` | [`kpod rmi`](./docs/kpod-rmi.1.md) |
-| `docker tag` | [`kpod tag`](./docs/kpod-tag.1.md) |
-| `docker pull` | [`kpod pull`](./docs/kpod-pull.1.md) |
-| `docker load` | [`kpod load`](TBD) |
-| `docker save` | [`kpod save`](TBD) |
-| `docker build` | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
-| `docker push` | [`buildah push`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-push.md) |
+| `docker build`  | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
+| `docker cp`     | [`kpod mount`](./docs/kpod-cp.md)        |
```
Comment (TomSweeneyRedHat) [inline transfer.md]: missing a '.1' in the link.  (.docs/kpod-cp.1.md)

Comment (rhatdan) [inline transfer.md]: fixed

File: README.md [hunk 5]
```diff
@@ -39,21 +39,24 @@ It is currently in active development in the Kubernetes community through the [d
...
-| [kpod-mount(1)](/docs/kpod-mount.1.md)     | Mount a working container's root filesystem |
-| [kpod-load(1)](/docs/kpod-load.1.md)       | Load an image from docker archive or oci |
-| [kpod-pull(1)](/docs/kpod-pull.1.md)       | Pull an image from a registry |
-| [kpod-push(1)](/docs/kpod-push.1.md)       | Push an image to a specified destination |
-| [kpod-rmi(1)](/docs/kpod-rmi.1.md)         | Removes one or more images   |
-| [kpod-save(1)](/docs/kpod-save.1.md)       | Saves an image to an archive |
-| [kpod-tag(1)](/docs/kpod-tag.1.md)         | Add an additional name to a local image |
-| [kpod-umount(1)](/docs/kpod-umount.1.md)   | Unmount a working container's root filesystem |
-| [kpod-version(1)](/docs/kpod-version.1.md) | Display the Kpod Version Information |
+| [crio(8)](/docs/crio.8.md)                           | Enable OCI Kubernetes Container Runtime daemon                                                       |
```
Comment (nalind) [inline README.md]: This isn't a new line, but `Enable` doesn't seem to belong there.

File: transfer.md [hunk 6]
```diff
@@ -39,11 +39,16 @@ There are other equivalents for these tools
...
-| `docker rmi` | [`kpod rmi`](./docs/kpod-rmi.1.md) |
-| `docker tag` | [`kpod tag`](./docs/kpod-tag.1.md) |
-| `docker pull` | [`kpod pull`](./docs/kpod-pull.1.md) |
-| `docker load` | [`kpod load`](TBD) |
-| `docker save` | [`kpod save`](TBD) |
-| `docker build` | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
-| `docker push` | [`buildah push`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-push.md) |
+| `docker build`  | [`buildah bud`](https://github.com/projectatomic/buildah/blob/master/docs/buildah-bud.md) |
+| `docker cp`     | [`kpod mount`](./docs/kpod-cp.1.md)       |
+| `docker images` | [`kpod images`](./docs/kpod-images.1.md)  |
```
Comment (nalind) [inline transfer.md]: This isn't in the right place if we're sorting alphabetically.

**Comment by nalind** [comment]
Nits, otherwise LGTM.

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat I think I have fixed everything, also added a link to the transfer page from the README.md.
I think we should merge and then continue cleanups in future pull requests.

**Comment by rhatdan** [comment]
@mrunalp Please merge, since this is documentation change only.  No reason to run tests.

**Pull request merged**